eng/scripts/Language-Settings.ps1

```diff
@@ -5,6 +5,50 @@ $packagePattern = "*.zip"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/python-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=python%2F&delimiter=%2F"
 
+function Get-AllPackageInfoFromRepo
+{
+  $allPackageProps = @()
+  Push-Location $RepoRoot
+  $allSetupProps = $null
+  try{
+    pip install packaging==20.4 -q -I
+    $allSetupProps = (python -c "import sys; import glob; import os; sys.path.append(os.path.join('scripts', 'devops_tasks')); from common_tasks import get_package_properties; \
```

Comment:
Have we validated that doing all of them in one python script call is more efficient?

Reply:
It is significantly faster than the previous method.
C:\Git\azure-sdk-for-python [PrepForToolsChanges]> Measure-Command { $props = Get-AllPackageInfoFromRepo }

Days              : 0
Hours             : 0
Minutes           : 0
Seconds           : 13
Milliseconds      : 149
Ticks             : 131495844
TotalDays         : 0.000152194263888889
TotalHours        : 0.00365266233333333 
TotalMinutes      : 0.21915974
TotalSeconds      : 13.1495844
TotalMilliseconds : 13149.5844